src/coreclr/nativeaot/System.Private.DisabledReflection/src/System.Private.DisabledReflection.csproj

```diff
@@ -1,4 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <CompileUsingReferenceAssemblies>false</CompileUsingReferenceAssemblies>
```

Comment:
Even though you explained this in the commit message, please add a comment to the projects in which you added this.